cmd/influx/main.go

```diff
@@ -205,7 +212,9 @@ func (c *CommandLine) connect(cmd string) {
 		fmt.Printf("Failed to connect to %s\n", c.Client.Addr())
 	} else {
 		c.Version = v
-		fmt.Printf("Connected to %s version %s\n", c.Client.Addr(), c.Version)
+		if !c.Dump {
```

Comment:
ah, I see it now.  Sometimes it's hard to spot the reasoning in these PR's as they only show parts of the code. I had to look at the entire file to understand the flow again.  My bad.